tokenizer.js

```diff
@@ -39,6 +39,7 @@ Tokenizer.prototype.configure = function (configuration) {
 
   if (Tokenizer.cache.contains(configuration.toString())) {
     this._regexes = Tokenizer.cache.get(configuration.toString());
+    this.alphanumeric();
```

Comment:
You're right, this is important to initialize the `Tokenizer`. But since it is not dependent on the if condition, shouldn't it be moved out of the if/else statement altogether?